pkg/services/ngalert/notifier/channels/alertmanager.go

```diff
@@ -96,6 +101,16 @@ func (n *AlertmanagerNotifier) Notify(ctx context.Context, as ...*types.Alert) (
 		return true, nil
 	}
 
+	_ = withStoredImages(ctx, n.logger, n.images,
+		func(index int, image *ngmodels.Image) error {
+			// If there is an image for this alert and the image has been uploaded
+			// to a public URL then include it as an annotation
+			if image != nil && image.URL != "" {
+				as[index].Annotations["image"] = model.LabelValue(image.URL)
```

Comment:
Can you make it a constant? We have a place where we put all system owned annotations/labels. Also, I wonder if we should add prefix `g_` to it. @gotjosh ?

Reply:
We can add `g_` but it won't be backwards compatible with Grafana 7 alerts. We can also add a constant, but just to give some additional context this annotation is specific to the Alertmanager notifier.

Reply:
Ok, that makes sense then.